Amplify/Categories/DataStore/Query/QueryOperator.swift

```diff
@@ -37,6 +38,10 @@ public enum QueryOperator {
                 return targetString.contains(predicateString)
             }
             return false
+        case .notContains(let predicateString):
+            if let targetString = target as? String {
+                return !targetString.contains(predicateString)
+            }
```

Comment:
Should we return `false` like for `.contains` above when the target is not a string?

Reply:
That would happen as is due to the default [`return false` underneath the switch statement](https://github.com/aws-amplify/amplify-swift/blob/68b61be6487a40669eb943f208f8874850c9a886/Amplify/Categories/DataStore/Query/QueryOperator.swift#L52). I don't think an explicit `return false` in this `case` block is necessary, but happy to add it if you think it provides value.

Reply:
Oh, I missed that, thanks for clarifying!